### PR TITLE
add indonesian

### PR DESCRIPTION
**Pull request by indonesia1331**
add indonesian id.json

### CODE REVIEW

**Review by indonesia1331 — COMMENTED** [review]
add indonesian id.json

**Comment by indonesia1331** [comment]
add indonesian

**Comment by elijahr2411** [comment]
yes we understand, you have opened and closed seven other pull requests at this point

**Comment by indonesia1331** [comment]
yeah

**Comment by indonesia1331** [comment]
indonesian language translate

**Comment by elijahr2411** [comment]
I'm not familiar with the indonesian language, are some of these sentences supposed to be lowercased?

```json

		"kAccountModal_NewPassword": "kata sandi baru",
		"kAccountModal_ConfirmNewPassword": "Konfirmasi password baru",
		"kAccountModal_CurrentPassword": "kata sandi saat ini",
		"kAccountModal_ConfirmPassword": "konfirmasi sandi",
```

**Comment by indonesia1331** [comment]
upercased 

		"kAccountModal_NewPassword": "Kata Sandi Baru",
		"kAccountModal_ConfirmNewPassword": "Konfirmasi Password Baru",
		"kAccountModal_CurrentPassword": "Kata Sandi Saat Ini",
		"kAccountModal_ConfirmPassword": "Konfirmasi Sandi",

File: static/lang/id.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		"kError_IncorrectPassword": "Kata kunci Salah.",
+
+		"kAccountModal_Verify": "Verifikasi email",
+		"kAccountModal_AccountSettings": "Pengaturan akun",
+		"kAccountModal_ResetPassword": "Atur Ulang Kata Sandi",
+
+		"kAccountModal_NewPassword": "kata sandi baru",
+		"kAccountModal_ConfirmNewPassword": "Konfirmasi password baru",
+		"kAccountModal_CurrentPassword": "kata sandi saat ini",
+		"kAccountModal_ConfirmPassword": "konfirmasi sandi",
```
Comment (indonesia1331) [inline static/lang/id.json]: update

**Comment by indonesia1331** [comment]
thank elijah

**Pull request merged**